etcdserver/etcdserverpb/rpc.proto

```diff
@@ -1000,6 +1000,8 @@ message StatusResponse {
   int64 dbSizeInUse = 9;
   // isLearner indicates if the member is raft learner.
   bool isLearner = 10;
+  // authRevision is the current revision of auth store
```

Comment:
I don't think this should be a field of StatusResponse. We already have a RPC `AuthStatus`. Could you move the field to the status of the RPC?

Reply:
Thanks for your advice, I will move this field to AuthStatus later. I add it to StatusResponse before because I didn't find AuthStatus at etcd v3.4, but just like @jingyih 's advice, metric may be a better choice at etcd v3.4.